### ISSUE TITLE
ECS latest version breaks website

### ISSUE THREAD

**Issue by VladimirNichovski** [dudaster/ele-custom-skin#753]
Hi!

I have a website where i have:
- ESC(v.3.1.4) 
- ESC Pro(v.3.1.0)
- Elementor(3.4.8)
- Elementor Pro(v.3.5.1).
With the last updates(both on ECS and Elementor) whenever i try to open a page where ECS is used i get the following message in console:
**Uncaught SyntaxError: Unexpected eval or arguments in strict mode**
And from then on every page i try to load is broken (no js is loaded at all).
Has anyone experienced this issue?
Can you help, Dev team?

**Comment by dudaster** [comment]
Please update everything to the latest version!

**Comment by VladimirNichovski** [comment]
Yes that is when the issue happens:  when i updated ECS, ECS Pro, Elementor and Elementor Pro to their latest versions the site breaks. With the versions stated above it works perfectly.

**Comment by dudaster** [comment]
Try to disable ECS first and then update the Elementor and afterwards update ECS and activate it.

This happens because from one version to another one library library changed its path.

**Comment by VladimirNichovski** [comment]
Thanks for the tip i will try this and let you know :)
